database/src/bin/sqlite-to-postgres.rs

```diff
@@ -193,6 +213,10 @@ impl Table for CollectorProgress {
         "aid, step, start, end"
     }
 
+    fn postgres_attributes() -> &'static str {
```

Comment:
Hmm, except for this one table, it seems like all the attribute names are the same. When trying to port this table, the script would crash, is that right?

If that is true, then we basically don't need `postgres_attributes`, right? Since if the names don't match, the code panics, and if they do match, we don't need the new method.

Reply:
Yes, this is the only table that has a different column name from sqlite.  But it wouldn't panic, because it just matches the column names in postgres.

It is also true that if we could ensure all column names are identical between the two databases, then we could just use one from `sqlite_attributes`.  The `postgres_attributes`  is only useful when postgres has a different column name from sqlite that represents the same content.

Reply:
Oh, sorry, I didn't realize that the `postgres_columns` logic fixes this. It's a bit complicated, but it works, so let's go with it.